print_docs.py

```diff
@@ -395,6 +395,10 @@ def cp(a, b):
   cp('nav.js', path+'nav.js')
   cp('searchWorker.js', path+'searchWorker.js')
 
+def copy_yaml_files(path):
+  for fn in ['100.yaml', 'undergrad.yaml', 'overview.yaml']:
+    shutil.copyfile(f'mathlib/docs/{fn}', path+fn)
```

Comment:
Actually, this is better:
```suggestion
    shutil.copyfile(f'{local_lean_root}docs/{fn}', path+fn)
```